cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -156,7 +156,8 @@ public synchronized RuntimeInfo lookup(Id.Program programId, final RunId runId)
       }
 
       // Program matched
-      RunRecordMeta record = store.getRun(programId, runId.getId());
+      // TODO: change what is passed in
```

Comment:
I'm going to remove this comment. Instead, added a comment here:
https://issues.cask.co/browse/CDAP-4121